website/docs/d/private_link_service_endpoint_connection_status.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service_endpoint_connection_status"
+sidebar_current: "docs-azurerm-datasource-private-link-service-endpoint-connection-status"
+description: |-
+  Use this data source to access endpoint connection information about an existing Private Link Service.
+---
+
+# Data Source: azurerm_private_link_service_endpoint_connection_status
+
+Use this data source to access endpoint connection information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service_endpoint_connection_status" "example" {
+  name                = azurerm_private_link_service.example.name
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+output "connection_name" {
+  value = data.azurerm_private_link_service_endpoint_connection_status.example.private_endpoint_connections.0.connection_name
+}
+
+output "private_endpoint_endpoint_name" {
+  value = data.azurerm_private_link_service_endpoint_connection_status.example.private_endpoint_connections.0.private_endpoint_name
+}
+
+output "private_endpoint_description" {
+  value = data.azurerm_private_link_service_endpoint_connection_status.example.private_endpoint_connections.0.description
+}
+
+output "private_endpoint_status" {
+  value = data.azurerm_private_link_service_endpoint_connection_status.example.private_endpoint_connections.0.status
+}
+
+output "private_endpoint_action_required" {
+  value = data.azurerm_private_link_service_endpoint_connection_status.example.private_endpoint_connections.0.action_required
+}
```

Comment:
I don't think we need this many example outputs

Reply:
Fixed.